.ci/jenkins/scripts/minikube.groovy

```diff
@@ -48,12 +48,19 @@ void waitForMinikubeStarted() {
     println 'Wait for Minikube components to be in Running state'
     def minikubeStatus = sh(returnStatus: true, script: '''
         set -x
+        function getKubeSystemPodStatus { kubectl get pods $1 -n kube-system -o json | jq '.items[].status.conditions[]'; }
+        function getKubeSystemPodReadyStatus { echo $(kubectl get pods $1 -n kube-system -o json | jq -r '.items[].status.conditions[] | select(.type == "Ready") | .status'); }
```

Comment:
Just an idea for future. What if we had these functions in a separate script which can be sourced?